### PR TITLE
Change KeyvaultKeyExistCV error to return -1 instead of 0

### PR DESCRIPTION
**Pull request by shayan-msft**
- Change KeyvaultKeyExistCV to return -1 in the error case to differentiate between error and key not existing
- Fix comment

### CODE REVIEW

**Review by jonahwilson — APPROVED** [review]

File: wrapper/cpp/main.go [hunk 1]
```diff
@@ -158,10 +158,10 @@ func KeyvaultKeyExistCV(serverName *C.char, groupName *C.char, keyvaultName *C.c
 	keys, err := keyClient.Get(ctx, C.GoString(groupName), C.GoString(keyvaultName), C.GoString(keyName))
 	if err != nil {
 		telemetry.EmitWrapperTelemetry("KeyvaultKeyExistCV", C.GoString(cv), err.Error(), "keyClient.Get", C.GoString(serverName))
-		return 0
+		return -1
 	}
 
-	// check the length and return 1 (means key exists) if there is more than one key
+	// check the length and return 1 (means key exists) if there is at least one key
```
[the remark this answers is not in the document]
Comment (shayan-msft) [reply wrapper/cpp/main.go:164]: Moc allows multiple keys per serverName, groupName, keyvaultName, keyName. keyClient.Get returns an array.
I'm not sure why though, maybe someone else can answer to that

[the remark this answers is not in the document]
Comment (SethBe) [reply wrapper/cpp/main.go:164]: At least 1 is correct, more then 1 key is unexpected, but possible.  IIRC when the key is rotate you will have multiple versions.  We disabled rotation in our use case, but we may enable it again in the future.

[the remark this answers is not in the document]
Comment (ncampuzano) [reply wrapper/cpp/main.go:164]: I see, but if we have multiple versions, those versions would be disabled and only one correct as you mentioned.

[the remark this answers is not in the document]
Comment (ncampuzano) [reply wrapper/cpp/main.go:164]: Thanks for the explanation. As this method is to test, the key exists, can imagine > instead of == would be fine.

**Review by msftpgayam — APPROVED** [review]

**Pull request merged**